config/vsc_hortense.py

```diff
@@ -0,0 +1,241 @@
+# ReFrame configuration file for VSC Tier-1 Hortense
+# https://docs.vscentrum.be/en/latest/gent/tier1_hortense.html
+#
+# authors: Sam Moors (VUB-HPC), Kenneth Hoste (HPC-UGent)
+
+from reframe.core.backends import register_launcher
+from reframe.core.launchers import JobLauncher
+
+account = "my-slurm-account"
+
+# use 'info' to log to syslog
+syslog_level = 'warning'
+
+perf_logging_format = 'reframe: ' + '|'.join([
+    'username=%(osuser)s',
+    'version=%(version)s',
+    'name=%(check_name)s',
+    'system=%(check_system)s',
+    'partition=%(check_partition)s',
+    'environ=%(check_environ)s',
+    'num_tasks=%(check_num_tasks)s',
+    'num_cpus_per_task=%(check_num_cpus_per_task)s',
+    'num_tasks_per_node=%(check_num_tasks_per_node)s',
+    'modules=%(check_modules)s',
+    'jobid=%(check_jobid)s',
+    '%(check_perfvalues)s',
+])
+
+format_perfvars = '|'.join([
+    'perf_var=%(check_perf_var)s',
+    'perf_value=%(check_perf_value)s',
+    'unit=%(check_perf_unit)s',
+]) + '|'
+
+
+@register_launcher('mympirun')
+class MyMpirunLauncher(JobLauncher):
+    def command(self, job):
+        return ['mympirun', '--hybrid', str(job.num_tasks)]
+
+
+site_configuration = {
+    'systems': [
+        {
+            'name': 'hortense',
+            'descr': 'Hortense',
+            'hostnames': ['login.*.dodrio.os'],
+            'modules_system': 'lmod',
+            'partitions': [
+                {
+                    'name': 'cpu_rome_256gb',
+                    'scheduler': 'slurm',
+                    'access': [f'-A {account} --export=NONE --get-user-env=60L --partition=cpu_rome'],
```

Comment:
Does this work for you? Because when I defined
```
['--constraint=something --export=None']
```
it was only respecting the `--constraint=something`. I had to split it in a list of access arguments, i.e. 
```
['--constraint=something', '--export=None']
```
I _think_ constraint might be a bit of specific case, I have the feeling it's parsed in a bit of a specific way [here](https://github.com/reframe-hpc/reframe/blob/fa813d0cd97a5aac4bedc24a7483f0f9abee4a75/reframe/core/schedulers/slurm.py#L218), which may have 'eaten' my `--export` argument. Whatever the case, the design idea of ReFrame _seems_ to be to specify a list of arguments, instead of putting them in one string. Maybe we can verify that with the devs, and if so, we should probably split this one up as well into a list of strings.

Reply:
@casparvl ReFrame doesn't do anything special in this case. it just adds a new `#SBATCH` line for each list item, which is perfectly fine as far as Slurm is concerned. 
but if `--constraint` is treated differently, then it would indeed be better for consistency to split it up.
```
#SBATCH -A badmin --export=NONE --get-user-env=60L --partition=gpu_rome_a100
```